### PR TITLE
fix: add favicon to extension pages (firefox)

### PR DESCRIPTION
**Pull request by im-adithya**
### Describe the changes you have made in this PR

Favicons in extension pages

### Link this PR to an issue [optional]

--

### Type of change

(Remove other not matching type)

- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes [optional]

Screenshot 2022-12-12 at 5 15 40 PM

### How has this been tested?

Manually

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Review by escapedcat — APPROVED** [review]
ack

**Comment by jankoegel** [comment]
@im-adithya looks good. should we add the same to `popup.html` and `prompt.html`!?
`popup.html` probably never appears in a tab of its own!?!
I don't know when `prompt.html` is rendered.

**Comment by escapedcat** [comment]
> `popup.html` probably never appears in a tab of its own!?!

Right, I don't think so.

> I don't know when `prompt.html` is rendered.

This is being prompted if you interact with a website i.e. "connect", "do a payment", etc.  
Does never show in a tab afaik

File: static/views/options.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 <head>
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="icon" type="image/png" href="assets/icons/alby_icon_yellow_32x32.png">
```
Comment (bumi) [inline static/views/options.html:7]: @escapedcat this path might be relevant for v3. - there was some change in the paths, wasn't it?

Comment (escapedcat) [reply static/views/options.html:7]: It should not matter for the metatags but will check

**Pull request merged**